### ISSUE TITLE
Game covers

### ISSUE THREAD

**Issue by BeemerDream** [GSTDiandre/ps2-discord-rpc#12]
Sorry to bother you again, but it doesn´t load any game covers. I read that it "pulls game covers from [Xlenore's PS2 Covers](https://github.com/xlenore/ps2-covers)"

**Comment by GSTDiandre** [comment]
Which version of OPL do you have? Did you try a different game? What game code are you having trouble with?

Also need screenshots of the command line prompt or logs.log

**Comment by BeemerDream** [comment]
Ok, I will provide screenshots later (I'm at work).
I use OPL v1.1.0, and I checked the site where it should pull the covers from and one of the games that I tested was Evil Dead and the cover was there, but didn't show up. I also tried 5-6 different games but none of them showed up. I will provide you with the game codes later. The PS2 logo showed up fine anyway when I was in OPL.

**Comment by BeemerDream** [comment]
So these are the games that I've tested:
SLUS_210.48
SCES_538.51
SLUS_210.75
SLUS_208.14
SLUS_207.77
SLUS_201.79
SLES_521.09
![PowerShell 7 (x64) 2024-10-01 21_48_26](https://github.com/user-attachments/assets/d707e38d-bd1c-49bf-9f53-954e4590b301)
![PowerShell 7 (x64) 2024-10-01 21_49_01](https://github.com/user-attachments/assets/8665c4ac-83f4-4d78-a7a0-4bae1b038624)
![Bilder 2024-10-01 22_02_21](https://github.com/user-attachments/assets/0a71f8a3-0060-43f2-b62c-53d2b85ef206)

**Comment by GSTDiandre** [comment]
I had fixed covers a while ago back when they stopped displaying due to Xlenore's repo adding 3D covers which changed the folder structure (/default/ was needed), but I failed to properly update the release. The updated code is already in `ps2rpc.py` in the main repo.

Download the latest release, it should work fine now.

I might also try and fix some cropping issues in the future since Discord has since changed how rich presence images are displayed over multiple updates, it now crops to show the top-most part of the cover instead of the center.
